aviatrix/resource_aviatrix_controller_bgp_max_as_limit_config.go

```diff
@@ -0,0 +1,91 @@
+package aviatrix
+
+import (
+	"context"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/terraform-providers/terraform-provider-aviatrix/goaviatrix"
```

Comment:
Our module path has been updated, please change to `github.com/AviatrixSystems/terraform-provider-aviatrix/v2/goaviatrix`

Reply:
Rebased with master and added this change.